rwengine/src/script/modules/GameModule.cpp

```diff
@@ -116,6 +116,9 @@ bool game_is_button_pressed(const ScriptArguments& args)
 
 void game_set_dead_or_arrested(const ScriptArguments& args)
 {
+	if (args.getWorld()->state->scriptOnMissionFlag == nullptr) {
+		return;
+	}
```

Comment:
This entire function doesn't seem correct at all. This just controls the behaviour of opcode 0112 within missions.